src/main/scala/viper/gobra/frontend/info/implementation/property/Assignability.scala

```diff
@@ -43,16 +44,22 @@ trait Assignability extends BaseProperty { this: TypeInfoImpl =>
     case (Single(lst), Single(rst)) => (lst, rst) match {
 
         // for go's types according to go's specification (mostly)
+      case (IntT(UntypedConst), r) if isIntegerType(underlyingType(r)) => true
+      case (l, IntT(UntypedConst)) if isIntegerType(underlyingType(l)) => true // not in spec
       case (l, r) if identicalTypes(l, r) => true
-      case (l, r) if !(l.isInstanceOf[DeclaredT] && r.isInstanceOf[DeclaredT])
-        && identicalTypes(underlyingType(l), underlyingType(r)) => true
+      // This rule seems to be ignored in the Go compiler
```

Comment:
What do you mean by that?

Reply:
The Go language spec states that
```
A value x is assignable to a variable of type T ("x is assignable to T") if one of the following conditions applies:
...
- x's type V and T have identical underlying types and at least one of V or T is not a defined type.
...
```

It is my understanding that, given such rule, the go compiler would accept something like
```go
package main

type IntType int

func main() {
	var x int = 1
	var y IntType = x
}
```
However, the compiler rejects this program with the message
```
./test.go:7:6: cannot use x (type int) as type IntType in assignment
```

Reply:
Then please make the comment more explicit. "Albeit stated in the language specification, the Go compiler does not accept assignments where ...."

Reply:
Fixed in b1738cf